===========================================================
#!/usr/bin/env python
-----------------------------------------------------------
try:
    a = {1,2,3}
    b = {3,4,5}
    print(a + b)
except TypeError: print("LOL")

------------------------------------------------------------
try:
    result = 10 / 0
except ZeroDivisionError:
    print("Division by zero is not allowed.")
Detail the structure of try, except, else, and finally blocks.

-------------------------------------------------------------
try:
    num = int(input("Enter a number: "))
except ValueError:
    print("Invalid input. Please enter a valid number.")
else:
    print("You entered:", num)
finally:
    print("Execution completed.")
Demonstrate handling of common exceptions.

---------------------------------------------------------------
file_name = "non_existent_file.txt"
try:
    with open(file_name, 'r') as file:
        content = file.read()
except FileNotFoundError:
    print(f"File '{file_name}' not found.")
    
---------------------------------------------------------------
# Создадим функцию для демонстрации примера
def example():

    # Бесконечный цикл, который продолжает выполняться  
    # до возникновения исключения
    while True:
        test4word = input("Как вас зовут? ")

        try:
            test4num = int(input("Сколько часов вы играете на своем мобильном телефоне?" ))

        # Если полученный ввод не число, будет вызвано исключение
        except ValueError:
            # Цикл будет повторяться до правильного ввода
            print("Error! Это не число, попробуйте снова.")

        # При успешном преобразовании в целое число,  
        # цикл закончится.
        else:
            print("Впечатляет, ", test4word, "! Вы тратите", test4num*60, "минут или", test4num*60*60, "секунд на игры в своем телефоне!")
            break

# Вызываем функцию
example()    

---------------------------------------------------------------------
while True:
    try:
        num1 = int(input("Input num1: "))
        num2 = int(input("Input num2: "))
        print(f"Division result is: {num1/num2}")
        break
    except ValueError:
        print("Wrong input. Try again.")
    except ZeroDivisionError:
        print("Division by zero is not allowed.")
        
        







